src/css/components/Input.scss

```diff
@@ -46,3 +47,11 @@ $form-input-bg-color: $cu-foreground--dark !default;
     display: none;
   }
 }
+
+// Hack to overrid webkit autofill styles
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus {
+  -webkit-box-shadow: 0 0 0px 1000px $cu-foreground--dark inset;
+  -webkit-text-fill-color: $cu-info;
+}
```

Comment:
What do you think about this @williamdingwall? It overrides that grody yellow background color that Chrome adds when fields are autofilled. I think this is a user preference, so it feels weird, but I don't ever even remember setting it, and it looks pretty bad with some of our themes and with our inset shadow.

Reply:
I'm cool with re-styling it.. yeah I doubt most people, or anyone except for  a veeeery small amount actually set that.